GitUI/Editor/Diff/ITextHighlightService.cs

```diff
@@ -22,4 +22,13 @@ public interface ITextHighlightService
     /// <param name="lineNumbersControl">The line number control</param>
     /// <param name="textEditor">The textEditor contol with text to adjust for.</param>
     void SetLineControl(DiffViewerLineNumberControl lineNumbersControl, TextEditorControl textEditor);
+
+    /// <summary>
+    /// Check if the index (line) in the text is a search match
+    /// for next/previous navigation, e.g. +/- for regular patches.
+    /// </summary>
+    /// <param name="lineNumbersControl">The line number control.</param>
+    /// <param name="indexInText">The index in the viewer text.</param>
+    /// <returns><see langword="true"/> if the line is a searchmatch; otherwise <see langword="false"/>.</returns>
+    bool IsSearchMatch(DiffViewerLineNumberControl lineNumbersControl, int indexInText);
```

Comment:
[nit] Could you please move it up and have it placed in alphabetical order?

Reply:
ok